test/controllers/confirm.company.controller.unit.ts

```diff
@@ -8,7 +8,7 @@ import { createConfirmationStatement } from "../../src/services/confirmation.sta
 import mocks from "../mocks/all.middleware.mock";
 import request from "supertest";
 import app from "../../src/app";
-import { CONFIRM_COMPANY_PATH } from "../../src/types/page.urls";
+import {CONFIRM_COMPANY_PATH, urlParams} from "../../src/types/page.urls";
```

Comment:
urlParams is unused and looks like lint didn't run as the {} need spaces around them. The githook should run when you commit and run the linter. Have you run this command in the confirmation-statement-web folder?
`find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;`

Reply:
That should setup the githook